client/src/lobby/createGame.ts

```diff
@@ -325,6 +349,32 @@ function submit() {
   closeAllTooltips();
 }
 
+function acceptOptionsFromGuest(data: GameOptions) {
+  // Table names are not saved
+  const name = $("#createTableName").val();
+
+  const options = {
```

Comment:
looks to violate dry, can't we reuse the Options interface or use the spread operator?